src/junox/lfo.js

```diff
@@ -1,18 +1,46 @@
 export default class LFO {
-  constructor ({ frequency, sampleRate }) {
-    this.frequency = frequency
-    this.sampleRate = sampleRate
+  constructor ({ frequency, delay = 0, sampleRate }) {
+    this.anchorRate = 1 / sampleRate
```

Comment:
@pendragon-andyh was there a specific reason you did use 1.2 ?
Because that indeed affects the LFO rate (making the anchor rate 20% faster)

see https://github.com/dzannotti/junox/pull/12/files#diff-9fe1faace2ee6567f6ea4f977e69b8f9R5 for functional testing